### DISCUSSION TITLE
Needs difficulty tweaking

### DISCUSSION THREAD

**Discussion by macxwdc** [divVerent/aaaaxy#10] [Spoilers]
Some of the levels are extremely difficult! And the camera jumps around alot, can there be an option to make the player always the center of the screen?  because its super difficult like this!

**Comment by macxwdc** [comment]
Oh yeah the pause menu is also annoying, i dont want to reset a level everytime i pause

**Comment by divVerent** [comment]
Camera rules cannot really be adjusted as "things not visible" are things that do not exist - multiple places in the game make use of this mechanic and thus a different camera control would be rather annoying or even make sections impossible without changing them around too. The camera is strongly inspired by Super Mario Maker's automatic camera system (i.e. camera lags behind you, and only follows your height while you're on ground). To be honest this doesn't work quite as well in a certain VVVVVV-inspired section which was intended as an optional secret area, so I may end up adjusting how the camera works for that section only.

Difficulty - indeed, the idea though is that all sections requiring difficult platforming are optional; you can beat the game without platforming skills, although this requires being creative in some places. However it is true that this fact is nowhere explained in the game, and I do not see a really good way to do so. In particular:

- Anti-Hub sections are hardest in the game.
- But only one of three must be beaten.
- Each focuses on different skill sets: left side is about being careful but does not require much of platforming skills. Bottom is a mix, containing however two rather difficult jumps that work kinda like a Mario-game shell jump and is otherwise rather easy on skill set. Right rather focuses on "regular" platforming, with the hardest part at the start.
- In the game end sections, no section requires strong platforming skills, although one section in particular _looks_ like it does. There is a way to "cheat the system" there, though :) as shown in some other places, looking up/down can be used to destroy/recreate objects that allow you to beat the room easily.

**Comment by divVerent** [comment]
As for pause menu, I intentionally do not want any "pausebuffering", i.e. pausing the game to give extra time to think. In exchange, lots of checkpoints are placed so that no room is overly long.

If you think a section is too long, it may be possible to add extra checkpoints there, though (e.g. I only recently split up the Driving You Mad section).

**Comment by divVerent** [comment]
I should actually say, initially I actually had a camera that was just attached to player center, no lagging behind etc. - but then on every jump everything scrolled around, which I found rather annoying.

Guess there isn't an ideal camera system. Sadly the way this game is designed, the camera system cannot be made toggle-able, though, as camera motion impacts gameplay rather intentionally.

**Comment by divVerent** [comment]
The section "The One Way Round" is also likely still much more difficult than it should be, even though it's easy once you know how. I adjusted hint texts and made the other exit - that leads towards a "secret" section - more hidden to try to lead players on the right path there.

**Comment by divVerent** [comment]
Made The Wide Gap substantially easier now; the third room of the sequence was unnecessarily hard.

That will however not be released until next week - I might tweak this more, and would prefer not releasing multiple times a week.

**Comment by divVerent** [comment]
Fixed an accidental "platform loss" in Little Big Platforms.

**Comment by divVerent** [comment]
I am going to close this discussion - all named concrete issues have been addressed, and for more issues, please open a new discussion thread. Thanks!
